### PR TITLE
Update env-variables.md

### PR DESCRIPTION
**Pull request by xuanhen2013**
feat: 增加 SOCKET_SERVER 的中文文档

-----
[View rendered docs/zh/guide/env-variables.md](https://github.com/xuanhen2013/umi/blob/patch-1/docs/zh/guide/env-variables.md)

---
This change is [Reviewable](https://reviewable.io/reviews/umijs/umi/3509)

### CODE REVIEW

**Comment by ycjcl868** [comment]
英文也加下

**Comment by xuanhen2013** [comment]
https://github.com/umijs/umi/pull/2032/commits/46854d3e56fd70ef3741784163e25cfd1e71b60f
英文2月份的时候已经有了

**Comment by xuanhen2013** [comment]
> 英文也加下

https://github.com/umijs/umi/pull/2032/commits/46854d3e56fd70ef3741784163e25cfd1e71b60f

File: docs/zh/guide/env-variables.md [hunk 1]
```diff
@@ -214,3 +214,10 @@ $ FORK_TS_CHECKER=1 umi dev
 ```bash
 $ USE_PUBLIC_IP=true umi dev
 ```
+
+### SOCKET_SERVER
+HMR 的 socket 服务, 默认为: window.location.host.
```
Comment (ycjcl868) [inline docs/zh/guide/env-variables.md]:
```suggestion
HMR 的 socket 服务, 默认为: window.location.host。
```

**Pull request closed without merging**